GhostNetwork.Publications.Domain/IPublicationStorage.cs

```diff
@@ -12,5 +12,7 @@ public interface IPublicationStorage
         Task<IEnumerable<Publication>> FindManyAsync(int skip, int take, IEnumerable<string> tags);
 
         Task<bool> UpdateOneAsync(string id, Publication publication);
+
+        bool FindeForbiddenWords(string content);
```

Comment:
Because of shared interface between publications and forbidden words I not be able to change sourse of forbidden words independently